specification/metrics/negotiatedcost.md

```diff
@@ -0,0 +1,34 @@
+# Negotiated Cost
+
+The Negotiated Cost represents a charge inclusive of the impacts of volume/tier-based rates and negotiated discounts while excluding any commitment-based discounts or the amortization of upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). The Negotiated Cost equals the [List Cost](#listcost) when no negotiated discounts are present. It is used to calculate savings based on negotiation activities, by comparing it with List Cost.
+
+The NegotiatedCost column MUST be present in the billing data. This column MUST be a valid numeric value of type Decimal and MUST NOT contain null values. When a [NegotiatedUnitPrice](#negotiatedunitprice) is not null, multiplying the NegotiatedUnitPrice by [QuantityInPricingUnit](#quantityinpricingunit ) MUST produce the NegotiatedCost. In cases where the NegotiatedUnitPrice is null, the following applies:
+
+* If the line item is based on other charges (e.g. [ChargeType](#chargetype) is 'Tax'), the NegotiatedCost MUST be calculated based on NegotiatedCost of the related charges
+* If the line item is unrelated to other charges (e.g. [AdjustmentType](#adjustmenttype) is 'Credit'), the NegotiatedCost MUST match the BilledCost.
```

Comment:
Unless the line item (=the charge, right?) relates to a commitment?

Reply:
The Negotiated Cost excludes the Commitments. The Billed Cost includes the Commitment.

Reply:
Yes!

Sorry, I was not clear.

I believe there are several cases for a line item that is unrelated to other charges, where the Negotiated Cost may be equal or different from the Billed Cost:
- Purchase of a commitment (whatever upfront, partial upfront, no upfront): Negotiated Cost = Billed Cost
- Usage of a commitment: Negotiated Cost != Billed Cost. The Billed Cost must be actually 0, since the cost resides in the commitment purchase record.
- Unused Commitment:  Negotiated Cost = Billed Cost.